app/src/main/java/no/usn/mob3000/data/model/content/FaqDto.kt

```diff
@@ -21,7 +21,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class FaqDto(
     @SerialName("id")
-    val faqId: String,
+    val faqId: String? = null,
```

Comment:
FAQ items will always have an ID. This change makes the tabulated data more difficult to work with and needs to be reverted.